src/averageAges.js

```diff
@@ -14,12 +13,24 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+
+function calculateAvarageAge(people) {
+  const ages = people.map(person => person.died - person.born);
+  const agesArr = ages.reduce((prev, person) => {
+    return prev + person;
+  }, 0) / ages.length;
```

Comment:
You may do the return statement right away. And pay attention to the naming, it should perfectly describe what is stored inside
```suggestion
  const averageAge = ages.reduce((prev, person) => prev + person, 0) / ages.length;
```